src/main/java/seedu/mark/ui/MainWindow.java

```diff
@@ -306,4 +328,97 @@ private CommandResult executeCommand(String commandText) throws CommandException
             throw e;
         }
     }
+
+    /**
+     * Compares two time in hours.
+     *
+     * @param before the time that is before.
+     * @param after the time that is after.
+     * @return the difference of two time in hour.
+     */
+    private long compareHour(LocalDateTime before, LocalDateTime after) {
+        return Duration.between(before, after).toHours();
+    }
+
+    /**
+     * Compares two time in minutes.
+     *
+     * @param before the time that is before.
+     * @param after the time that is after.
+     * @return the difference of two time in minute.
+     */
+    private long compareMinute(LocalDateTime before, LocalDateTime after) {
+        return Duration.between(before, after).toMinutes();
+    }
+
+    /**
+     * Creates a notification for a specific reminder.
+     *
+     * @param reminder the reminder that is used to create notification.
+     * @return the notification for reminder.
+     */
+    private Notifications getNoti(Reminder reminder) {
+        Notifications noti = Notifications.create();
```

Comment:
Suggesting a more common abbreviation for `notification`:
```suggestion
        Notifications notif = Notifications.create();
```

Reply:
ok